windows-pod-security/restricted/require-run-as-containeruser/require-run-as-containeruser.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-run-as-containeruser
+  annotations:
+    policies.kyverno.io/title: Require Run As ContainerUser
+    policies.kyverno.io/category: Pod Security Standards (Restricted)
```

Comment:
This is not in the official [restricted profile](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted). Please use a different category ensuring you also update Artfact Hub metadata.

Reply:
Hi @chipzoller 
runAsUserName in Windows is equivalent to runAsUser in Linux.

The following is the restricted profile:

Running as Non-root user (v1.23+):
Containers must not set runAsUser to 0.
Restricted Fields:
spec.securityContext.runAsUser
spec.containers[*].securityContext.runAsUser
spec.initContainers[*].securityContext.runAsUser
spec.ephemeralContainers[*].securityContext.runAsUser
Allowed Values:
Any non-zero value
Undefined/null
That is the reason why I categorized this policy under Pod Security Standards (Restricted). However, it is not in the official documentation.

Could you please suggest any category for this policy?

Reply:
I understand, but it isn't part of the formal [Pod Security Standards definitions](https://kubernetes.io/docs/concepts/security/pod-security-standards/#profile-details) so it doesn't belong in that category. How about a category called "Windows Security"?

Reply:
Windows Security looks good, i will update the chanages